### PR TITLE
Completed Hero section

### LINKED ISSUE #9: Create Hero component for home page
## Summary

Create a Hero section for the Home page with a search bar in it. This should encourage users to start using the website and showcase the website's usage.

### Screenshots
This is only an example image of a previous project. Feel free to change the design.
image

### Basic example
N/A

### Motivation

We want users to be able to use the website's main feature quickly and show them what we offer

### PR DESCRIPTION
**Pull request by Imanzuher**
this PR closes #9

### CODE REVIEW

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+function Hero() {
```
Comment (AllanSaleh) [inline src/components/Hero.jsx:1]: Awesome work on writing clean code!!🎊
It looks great!!✨

File: tailwind.config.js [hunk 2]
```diff
@@ -3,15 +3,24 @@ module.exports = {
...
+        inter: ['Inter'],
+      },
+      width: {
+        500: '500px',
+        300: '300px',
+      },
+      screens: {
+        small: { min: '360px', max: '650px' },
+        medium: { min: '651px', max: '1023px' },
+        large: { min: '1024px', max: '1279px' },
```
Comment (AllanSaleh) [inline tailwind.config.js:22]: Superb work on adding these screen sizes here!🙌
This is going to help the whole team use these right away!🔥

**Review by AllanSaleh — CHANGES_REQUESTED** [review]
Beautiful work on creating this component!🎊
It looks flawless! Also, great work on making it responsive!👏

The only thing that is left is to make a snapshot for this component.
You can find out how to make one [here](https://jestjs.io/docs/snapshot-testing). 
Other than that, you are doing AMAZING!

Let me know if you need any help. :)

File: tailwind.config.js (hunk 2 above)
Comment (Imanzuher) [reply tailwind.config.js:22]: Thank you Allan

**Comment by AllanSaleh** [comment]
Amazing work!🙌

**Review by AllanSaleh — APPROVED** [review]
Amazing work!🙌
LGTM!👏

**Pull request merged**